### PR TITLE
Write more markdown, less html

### PR DESCRIPTION
**Pull request by asottile**
I for one keep making html mistakes!

This swaps bits of the html out for inline markdown, which imo is both easier to _write_ and _edit_.

I only did some parts of the file -- if this seems good I'll do more of the file and add a few features to the markdown parser such that I can eliminate even more html blocks.

### CODE REVIEW

**Comment by asottile** [comment]
CC @Lucas-C @chriskuehl

**Comment by chriskuehl** [comment]
html and markdown are both such legacy technologies. personally i think it'd be less error-prone if you re-wrote this using react components.

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,6 @@
 *.html
-*.iml
+*.pyc
 *.swp
-.DS_Store
-.idea
-.project
-.pydevproject
```
Comment (chriskuehl) [inline .gitignore]: +++

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 aspy.yaml
 mako
 libsass
+markdown-code-blocks
```
Comment (chriskuehl) [inline requirements-dev.txt]: nice

**Pull request merged**